ui/src/templates/components/CommunityTemplateInstallerOverlay.tsx

```diff
@@ -67,15 +58,11 @@ class CommunityTemplateInstallerOverlayUnconnected extends PureComponent<
             onDismiss={this.onDismiss}
           />
           <Overlay.Body>
-            <Panel border={true} gradient={Gradients.RebelAlliance}>
-              <Panel.Header>
-                <Heading element={HeadingElement.H3}>{templateName}</Heading>
-              </Panel.Header>
-              <Panel.Body>
-                Installing this template will create {resourceCount} resources
-                in your system
-              </Panel.Body>
-            </Panel>
+            <CommunityTemplateName
+              templateName={templateName}
+              resourceCount={resourceCount}
+              // onClickInstall={() => {}}
```

Comment:
If `onClickInstall` has a function passed in the green install button will appear. Made it optional so it won't be rendered until the function is ready, at which point we should probs make this prop required